nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -186,7 +186,10 @@ def init_model(self, cfg: DictConfig, trainer: Trainer):
         self.prompt_encoder = None
 
     def first_stage_of_pipeline(self):
-        return self.frozen_model.model.pre_process
+        if self.megatron_amp_o2:
```

Comment:
Should you check if O2 True/False is consistent between the frozen model and your prompt generator?

Reply:
Good question, will double check.